File: CS/Shared/Northwind/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace DevExpress.CRUD.Northwind {
    public class Category {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public virtual ICollection<Product> Products { get; set; }
    }
}

File: VB/GridControlCRUD.Common.Northwind/Northwind - VB/Category.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace DevExpress.CRUD.Northwind
	Public Class Category
		Public Property Id() As Long
		Public Property Name() As String
		Public Property Description() As String
		Public Overridable Property Products() As ICollection(Of Product)
	End Class
End Namespace
